app/components/channel_drawer_list/channel_drawer_list.js

```diff
@@ -361,6 +362,27 @@ class ChannelDrawerList extends Component {
         );
     };
 
+    sectionAction = (action) => {
```

Comment:
Can you rename this to `renderSectionAction` to be more consistent with other components?